Neos.ContentGraph.DoctrineDbalAdapter/src/Domain/Repository/ContentGraph.php

```diff
@@ -137,9 +137,10 @@ public function findRootNodeAggregates(
         FindRootNodeAggregatesFilter $filter,
     ): NodeAggregates {
         $queryBuilder = $this->createQueryBuilder()
-            ->select('n.*, h.contentstreamid, h.name, h.dimensionspacepoint AS covereddimensionspacepoint')
+            ->select('n.*, h.contentstreamid, h.name, dsp.dimensionspacepoint AS covereddimensionspacepoint')
```

Comment:
IIRC we added a comment explaining the table aliases on top of this class. Could you add `dsp`, `pdsp` and `cdsp` to the list?